lib/externalGraderResults.js

```diff
@@ -101,6 +101,11 @@ function loadQueueUrl(sqs, callback) {
     }
 }
 
+function logAtTimeout(jobId, logBundle) {
+    logger.info('externalGradingResults timeout logs for ' + jobId);
+    logger.info(logBundle);
```

Comment:
Logger statements using winston should always have a string as the first argument. I suggest upgrading this to an error and replacing the two lines above with:
```
logger.error('externalGradingResults timeout', {jobId, logBundle});
```